flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/utils/StatementUtils.java

```diff
@@ -321,4 +321,10 @@ private static String buildSelectWithBoundaryRowLimits(
     private static String quotedTableIdString(TableId tableId) {
         return tableId.toQuotedString('`');
     }
+
+    private static String quotedFieldString(String field) {
```

Comment:
This method is duplicated as `StatementUtils#quote`. We could use `quote` directly.

Reply:
Okay, let me reprocess this.